cohttp/src/jbuild

```diff
@@ -2,7 +2,7 @@
 #require "unix"
 
 let version =
-  let ic = open_in "../VERSION" in
+  let ic = open_in "../../VERSION" in
```

Comment:
there's got to be a better way to do version stamping, but not a blocker to merge :-)